protected/controllers/PatientController.php

```diff
@@ -1845,14 +1845,16 @@ public function actionPracticeList($term)
         $criteria->addCondition('( (date_end is NULL OR date_end > NOW()) AND (date_start is NULL OR date_start < NOW()))');
         
         $criteria->addSearchCondition('LOWER(CONCAT_WS(", ", address1, address2, city, county, postcode))', $term);
+        $criteria->addSearchCondition('first_name', $term, true, 'OR');
```

Comment:
Should the first name be LOWER()'d to provide case insensitive matching?

Reply:
Yes. Good point.